frontend/src/components/user-management/TeacherUserTable.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { User } from "../../types/UserTypes";
+import { TableRow, Table } from "../common/Table";
+import RemoveUserPopover from "./RemoveUserPopover";
+
+interface TeacherUserTableProps {
+  teacherUsers: User[];
+}
+
+const TeacherUserTable = ({
+  teacherUsers,
+}: TeacherUserTableProps): React.ReactElement => {
+  const headers = ["Name", "Email"];
```

Comment:
It shouldn't be identical to `AdminTable`! it should also have a `School` field. Please see design for more details!

Reply:
looks like the designs have a `Region` field instead of `School` - can you confirm ?

Reply:
Oops sorry I missed that

Reply:
Wait is there a good way of getting school data for the user? I don't think the user model has a school field

Reply:
you may want to define a new function in the backend (i.e. `getSchoolByTeacherId`)

Reply:
You can also define a new function that returns all the teacher data with the respective school
-> prob better to keep it in the backend